website/content/en/docs/crds-scope.md

```diff
@@ -0,0 +1,94 @@
+---
+title: CRD scope with Operator SDK
+linkTitle: CRD Scope
+weight: 60
+---
+
+## Overview
+
+The CustomResourceDefinition (CRD) scope can also be changed for cluster-scoped operators so that there is only a single 
+instance (for a given name) of the CRD to manage across the cluster.
+
+> **NOTE**: Cluster-scoped CRDs are **NOT** supported with the Helm operator. While Helm releases can create 
+>cluster-scoped resources, Helm's design requires the release itself to be created in a specific namespace. Since the 
+>Helm operator uses a 1-to-1 mapping between a CR and a Helm release, Helm's namespace-scoped release requirement 
+>extends to Helm operator's namespace-scoped CR requirement.
+
+For each CRD that needs to be cluster-scoped, update its manifest to be cluster-scoped.
+
+* `deploy/crds/<full group>_<resource>_crd.yaml`
+  * Set `spec.scope: Cluster`
+
+To ensure that the CRD is always generated with `scope: Cluster`, add the tag 
+`// +kubebuilder:resource:path=<resource>,scope=Cluster`, or if already present replace `scope={Namespaced -> Cluster}`, 
+above the CRD's Go type definition in `pkg/apis/<group>/<version>/<kind>_types.go`. Note that the `<resource>` 
+element must be the same lower-case plural value of the CRD's Kind, `spec.names.plural`. 
+
+## CRD cluster-scoped usage 
+
+This scope is ideal for the cases where an instance(CR) of some Kind(CRD) will be used in more than one namespace 
+instead of a specific one. 
+
+> **NOTE**: When a `Manager` instance is created in the `main.go` file, it receives the namespace(s) as Options. 
+>These namespace(s) should be watched and cached for the Client which is provided by the Controllers. Only clients 
+>provided by cluster-scoped projects where the `Namespace` attribute is `""` will be able to manage cluster-scoped CRD's. 
+>For more information see the [Manager][manager_user_guide] topic in the user guide and the 
+>[Manager Options][manager_options].
+
+## Example for changing the CRD scope from namespace to cluster 
+
+The following example is for Go based-operators. Note that for Helm and Ansible based-operators the changes in the CRD 
```

Comment:
In the docs the standard is Type based-operator because of this I will not address `Go-based operators`. See the changelog for example. 

It is an old doc and text which would not be part of the scope of this PR, however, why not address your other suggestions :-P

Reply:
I see `based-operator` in 21 places, but `-based operator` in 32 places. The latter is grammatically correct too.

Reply:
HI @estroz, 

It is no part of the scope of this PR also I am not sure since I quite remember that I started t use Ansible based-operators after some reviews made by Joe. 

So, IMO if it is wrong is before the changes made here and should not be a blocker for we move forward any away.